src/main/java/pro/sky/Coursework2/controller/ExamController.java

```diff
@@ -1,28 +1,27 @@
 package pro.sky.Coursework2.controller;
 
+import org.springframework.web.bind.annotation.*;
 import pro.sky.Coursework2.model.Question;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestParam;
-import org.springframework.web.bind.annotation.RestController;
 
 
+import pro.sky.Coursework2.service.ExaminerService;
 import pro.sky.Coursework2.service.ExaminerServiceImpl;
 
 import java.util.Collection;
 
 @RestController
 @RequestMapping("/get")
 public class ExamController {
-    private final ExaminerServiceImpl examinerService;
+    private final ExaminerService service;
 
 
-    public ExamController(ExaminerServiceImpl examinerService) {
-        this.examinerService = examinerService;
+    public ExamController(ExaminerService service) {
+
+        this.service = service;
     }
 
-    @GetMapping
-    public Collection<Question> getQuestion(@RequestParam int amount) {
-        return examinerService.getQuestion(amount);
+    @GetMapping()
```

Comment:
```suggestion
    @GetMapping("/{amount}")
```
иначе работать не будет, надо указать, где в URL будет параметр находиться